src/node_constants.h

```diff
@@ -27,6 +27,9 @@
 #include "node.h"
 #include "v8.h"
 
+#define EXTENSIONLESS_FORMAT_JAVASCRIPT (0)
+#define EXTENSIONLESS_FORMAT_WASM (1)
```

Comment:
AFAICT these are going to be exposed as enumerable properties of `fs.constants` in userland?